oap-server/server-library/library-datacarrier-queue/pom.xml

```diff
@@ -29,8 +29,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.openjdk.jmh</groupId>
-            <artifactId>jmh-generator-annprocess</artifactId>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>server-testing</artifactId>
+            <version>${project.version}</version>
```

Comment:
I think `LinkedArrayBenchmark` has been moved into `microbench` module already. We don't remove this dependency, and existing test case should be removed too, right?

Reply:
Same requests for other repositories. There should be no `pom.xml` change about `jmh`->`server-testing`, right?

Reply:
Yes I have removed this, thank you for checking